sdk/typescript/runtime/main.go

```diff
@@ -253,7 +276,41 @@ func (t *TypescriptSdk) Base() (*dagger.Container, error) {
 	}
 }
 
-// setupModule initialiaze the user's module.
+// addTemplate adds the template files to the user's module if there's no
+// source files in the src directory.
+func (t *TypescriptSdk) addTemplate(ctx context.Context, ctr *dagger.Container) (*dagger.Container, error) {
```

Comment:
I don't understand why we're doing this?

Shouldn't `dagger init` have done this -- and if not, just fail at this point?

Reply:
Dagger init is calling `Codegen`

Reply:
See #8201 for how I want to handle this in the future - but yes, atm, there's no explicit init step, it needs to all get done in `Codegen` (which leads to all sorts of bugs, hence #8201).